f-pms/UI
# feat(PMS-41): Web socket connect example

Description: Create a webSocketStore => connect/disconnect to ws on app start/stop => create 2 buttons for subscribe and unsubscribe to ws's channel on the Dashboard page
Step to verify the result:

1. Start the PLC via this tutorial: https://hbc-capstone.atlassian.net/l/cp/PAhB823s
2. Start back-end (watch demo) verify if there are logs on the console
3. Start front-end and navigate to Dashboard page.

Demo: https://github.com/f-pms/UI/assets/59760568/fe14e67d-1d67-45e1-af24-fa3deaba247d
